### ISSUE TITLE
Certain lang file sections are unmodifiable [Fabric Waystones 2.2.3 || 1.17]

### ISSUE THREAD

**Issue by JuceDoesThings** [LordDeatHunter/FabricWaystones#91]
Currently, I am trying to make this mod work well with a dark GUI. Certain parts, however, can't be modified in color no matter how hard you try. Things like this, for example, don't work:

{
"waystones.config.owner": "\u00a7r\u00a7fOwned by %s"
}

It causes weird issues, I'll show some of them here
![image](https://user-images.githubusercontent.com/85078618/148688641-bb8e66b5-4571-46f0-98ae-0a89fae6bb87.png)
![image](https://user-images.githubusercontent.com/85078618/148688647-31f2af8f-0a3b-45fd-a4ca-4daab19ebdb5.png)

Not to mention the fact that the names just cannot be changed in color
![image](https://user-images.githubusercontent.com/85078618/148688670-61f05338-5bc1-4851-b47f-19254d9e190e.png)

I don't see this being high priority, but you making some argument color for those would be really helpful if you could!

**Comment by JuceDoesThings** [comment]
This is technically more of an enhancement thing than anything, I just thought it'd be nice. A way to solve this, I think, is to have an argument color for all of these things so that they can be modified the same

**Comment by LordDeatHunter** [comment]
I've already implemented most of this in the latest (1.18) version. I only go back to older versions for more critical issues, and if there's a high enough demand.

**Comment by JuceDoesThings** [comment]
Ah shoot, okay. Sorry, I just can't use 1.18 yet so I was posting. That's understandable

**Comment by LordDeatHunter** [comment]
Any particular reason for that? I think most older mods have been updated, as well as the optimization mods.

**Comment by JuceDoesThings** [comment]
Yeah! My graphics card is super old and already struggles when chunk loading in 1.17, even stuff like Sodium can't help me with 1.18 chunk loading. The load is too intense and just absolutely tanks fps any time I make the mistake of moving. It's fine, though, I'll just remember to try and get this fixed when I eventually get a newer graphics card and update to 1.18
